### PR TITLE
Create technische-universitat-dresden-betriebswirtschaftslehre-rechnungswesen-controlling.csl

### PR DESCRIPTION
**Pull request by bwiernik**
https://forums.zotero.org/discussion/84992/style-request-american-psychological-association-7th-edition-tu-dresden

### CODE REVIEW

**Comment by adam3smith** [comment]
I couldn't open the style guidelines, but I'm guessing this should have default-locale set to German?

**Comment by bwiernik** [comment]
I think so. I'm waiting for that and a few other questions from the requester.

File: technische-universitat-dresden-betriebswirtschaftslehre-rechnungswesen-controlling.csl [hunk 1]
```diff
@@ -0,0 +1,1655 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" page-range-format="expanded">
+  <info>
+    <title>Technische Universität Dresden - Betriebswirtschaftslehre/Rechnungwesen/Controlling (APA)</title>
```
Comment (rmzelle) [inline technische-universitat-dresden-betriebswirtschaftslehre-rechnungswesen-controlling.csl:4]: I'm also wondering if this should be renamed to "... - BWL, insb. Betriebliches Rechnungswesen/Controlling (APA)", if this style is for that department (I don't see any hits on Google for the verbatim "Betriebswirtschaftslehre/Rechnungwesen/Controlling").

Comment (bwiernik) [reply technische-universitat-dresden-betriebswirtschaftslehre-rechnungswesen-controlling.csl:4]: Fair enough. I was just being consistent with the other Dresden styles

Comment (rmzelle) [reply technische-universitat-dresden-betriebswirtschaftslehre-rechnungswesen-controlling.csl:4]: Ah, right, let's keep it like this, then.

**Comment by rmzelle** [comment]
@bwiernik, let me know if I can merge this as-is, or if you'd like to tackle the outstanding requests in the forum thread.

**Comment by bwiernik** [comment]
Okay, I've updated the issues from the thread. Should be good to merge now. I can make other changes later if needed.

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**